src/user.py

```diff
@@ -0,0 +1,141 @@
+import logging
+import json
+import uuid
+from datetime import datetime
+import logging
+from google.auth import jwt
+from flask import Blueprint, jsonify, request
+from src.constants.http_status_codes import HTTP_200_OK,HTTP_201_CREATED, HTTP_400_BAD_REQUEST, HTTP_401_UNAUTHORIZED
+from src.models import UserLogin, UserProfile, db, Borrowing
+
+
+user = Blueprint('user', __name__, url_prefix='/api/v1/user')
+
+
+def decode_token(token_object):
+     
+    """
+    Decodes a google token and returns a JSON object with the following details
+    {
+        "at_hash": "ekLIO2gVVwjiH02eiu88hQ",
+        "aud": "170044535313-f02c8hd020ptha7n0t4fraouo2ap9bqq.apps.googleusercontent.com",
+        "azp": "170044535313-f02c8hd020ptha7n0t4fraouo2ap9bqq.apps.googleusercontent.com",
+        "email": "email@mail.com",
+        "email_verified": true,
+        "exp": 1636222353,
+        "family_name": "Last_Name",
+        "given_name": "First_Name",
+        "iat": 1000000000,
+        "iss": "accounts.google.com",
+        "jti": "0019b0aaa2e355f8fa2b2ce8b3bbbbab7b63a5014",
+        "locale": "en-GB",
+        "name": "First_Name Name",
+        "picture": "https://lh3.googleusercontent.com/a-/image_url..",
+        "sub": "113501893650341726537"
+    }
+    """
+    # try:
+    #     return jwt.decode(token_object, verify=False)
+    # except Exception as ex:
+    #     logging.critical(ex)
+    return jwt.decode(token_object, verify=False)
```

Comment:
Maybe a small comment on why `verify=False` in case someone else without much context has to work with this code?